test/unit/host_ansible_role_test.rb

```diff
@@ -17,4 +17,25 @@ class HostAnsibleRoleTest < ActiveSupport::TestCase
     end
     should validate_uniqueness_of(:ansible_role_id).scoped_to(:host_id)
   end
+
+  describe 'auditing' do
+    test 'audit creation and deletion of host ansible roles' do
```

Comment:
I recommend splitting this into separate tests for each action. One test should focus on verifying the audit record for creation, and another for deletion.